ggshield/cmd/sca/scan/ci.py

```diff
@@ -0,0 +1,75 @@
+from pathlib import Path
+from typing import Any, Optional, Sequence
+
+import click
+
+from ggshield.cmd.common_options import directory_argument
+from ggshield.cmd.sca.scan.sca_scan_utils import (
+    create_output_handler,
+    display_sca_beta_warning,
+    sca_scan_all,
+    sca_scan_diff,
+    update_context,
+)
+from ggshield.cmd.sca.scan.scan_common_options import add_sca_scan_common_options
+from ggshield.cmd.utils import get_commits
+from ggshield.core.errors import handle_exception
+from ggshield.sca.collection.collection import (
+    SCAScanAllVulnerabilityCollection,
+    SCAScanDiffVulnerabilityCollection,
+)
+
+
+@click.command()
+@add_sca_scan_common_options()
+@click.pass_context
+@directory_argument
+@click.option(
+    "--all",
+    help="Scan all files.",
+    is_flag=True,
+)
```

Comment:
I don't really get this one, why not just launching `sca scan all` in that case ?

Reply:
It's what has been defined in [the specs](https://www.notion.so/gitguardian/IaC-SCA-ggshield-CLI-options-24f4a38da7294e858778723ce14cb414?pvs=4#22e76b00ea484f4bb0d28e235c717af9).

Reply:
Indeed, this will have the same behavior.
We thought it was easier for the users to always use the same command in the CI except a flag rather than having two different commands.

Reply:
Ok thanks